extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -211,11 +212,15 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 	/**
 	 * Gets the Database Schema Provider version for a SQL project
 	 */
-	async getProjectDatabaseSchemaProvider(projectFilePath: string): Promise<string> {
+	public async getProjectDatabaseSchemaProvider(projectFilePath: string): Promise<string> {
 		return await this.projectController.getProjectDatabaseSchemaProvider(projectFilePath);
 	}
 
-	async generateProjectFromOpenApiSpec(options?: sqldbproj.GenerateProjectFromOpenApiSpecOptions): Promise<sqldbproj.ISqlProject | undefined> {
+	public async generateProjectFromOpenApiSpec(options?: sqldbproj.GenerateProjectFromOpenApiSpecOptions): Promise<sqldbproj.ISqlProject | undefined> {
```

Comment:
why is public getting added to only these two functions? Aren't these all public by default?

Reply:
Will follow up with adding the rest. It's true they're public by default, but I prefer being explicit 😄